pkg/webui/account/components/oauth-client-form/index.js

```diff
@@ -0,0 +1,346 @@
+  )
+
+  const initialValues = {
+    owner_id: userId,
+    rights: [...pseudoRights],
+    ...values,
+  }
+
+  return (
+    <Form
+      error={error}
+      onSubmit={handleSubmit}
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+    >
+      <Form.Field
+        title={sharedMessages.oauthClientId}
+        name="ids.client_id"
+        placeholder={m.clientIdPlaceholder}
+        component={Input}
+        disabled={update}
+        autoFocus={!update}
+        required
+      />
+      <Form.Field
+        title={sharedMessages.name}
+        name="name"
+        placeholder={m.clientNamePlaceholder}
+        component={Input}
+      />
+      <Form.Field
+        title={sharedMessages.description}
+        type="textarea"
+        name="description"
+        placeholder={m.clientDescPlaceholder}
+        description={m.clientDescDescription}
+        component={Input}
+      />
+      <Form.Field
+        name="redirect_uris"
+        title={m.redirectUrls}
+        valuePlaceholder={m.urlsPlaceholder}
+        addMessage={m.addRedirectUri}
+        component={KeyValueMap}
+        indexAsKey
+        description={m.redirectUrlDescription}
+      />
+      <Form.Field
+        name="logout_redirect_uris"
+        title={m.logoutRedirectUrls}
+        valuePlaceholder={m.urlsPlaceholder}
+        addMessage={m.addLogoutRedirectUri}
+        component={KeyValueMap}
+        description={m.logoutRedirectUrlsDescription}
+        indexAsKey
+      />
+      {isAdmin && (
+        <>
+          <Form.Field
+            title={sharedMessages.state}
+            name="state"
+            component={Select}
+            options={approvalStateOptions}
+          />
+          <Form.Field
+            title={sharedMessages.stateDescription}
+            name="state_description"
+            component={Input}
+            type="textarea"
+            placeholder={m.userDescPlaceholder}
+          />
+          <Form.Field
+            title={m.skipAuthorization}
+            name="skip_authorization"
+            component={Checkbox}
+            description={m.skipAuthorizationDesc}
+            disabled={!isAdmin}
+          />
+          <Form.Field
+            title={m.endorsed}
+            name="endorsed"
+            component={Checkbox}
+            description={m.endorsedDesc}
+            disabled={!isAdmin}
+          />
+        </>
+      )}
+      {((isAdmin && update) || !update) && (
+        <Form.Field
+          title={m.grants}
+          name="grants"
+          encode={encodeGrants}
+          decode={decodeGrants}
+          component={Checkbox.Group}
+          description={m.grantsDesc}
+        >
+          <Checkbox name="GRANT_AUTHORIZATION_CODE" label={m.grantAuthorizationLabel} />
+          <Checkbox name="GRANT_REFRESH_TOKEN" label={m.grantRefreshTokenLabel} />
+          <Checkbox name="GRANT_PASSWORD" label={m.grantPasswordLabel} disabled={!isAdmin} />
```

Comment:
```suggestion
          {isAdmin && <Checkbox name="GRANT_PASSWORD" label={m.grantPasswordLabel} />}
```
Let's not even show the option.